UnityProject/Assets/Scripts/Items/PDA/PDALogic.cs

```diff
@@ -356,6 +359,7 @@ private void ServerInsertItem(GameObject item, ItemSlot fromSlot, GameObject pla
 						$"After a moment it disappears, your Telecrystal counter ticks up a second later";
 
 					Chat.AddExamineMsgFromServer(player, uplinkMessage);
+					PDAGui.uplinkPage.UpdateTCCounter();
```

Comment:
Could this null if you spawn a PDA in and then use a telecrystal without first opening the UI?

Reply:
no, its set in the prefab of the tab

Reply:
But the tab wont be created until it has been opened the first time

Reply:
yeah whoops